platform/core/src/log.js

```diff
@@ -4,8 +4,23 @@ const log = {
   info: console.log,
   trace: console.trace,
   debug: console.debug,
-  time: console.time,
-  timeEnd: console.timeEnd,
+  time: key => {
+    log.timingKeys[key] = true;
+    console.time(key);
+  },
+  timeEnd: key => {
+    if (!log.timingKeys[key]) {
```

Comment:
can we consle.warn here before retun that key does not exist?

Reply:
That is specifically avoided - we get lots more time ends than we do time starts, and we don't care that the key doesn't exist any longer.